core/string/ustring.cpp

```diff
@@ -39,7 +39,7 @@
 #include "core/string/translation_server.h"
 #include "core/string/ucaps.h"
 #include "core/variant/variant.h"
-#include "core/version_generated.gen.h"
+#include "core/version.h"
```

Comment:
Not strongly opposed to it, but why that change? The only macro used is from `version_generated.gen.h` so it's more specific.

Reply:
My thinking was the `export` pragma in `version.h` makes it so that file is "exposing" the generated file, so it can be used in its place similar to using `typedef.h` instead of specific `std` headers. That's not as easily conveyed though, so I'm fine reverting back to the generated file.

Reply:
Well it's not a bad idea, and the cost on compilation time should be similar either way.